docs/sources/operations/storage/schema/_index.md

```diff
@@ -33,19 +60,19 @@ Here are items to consider when changing the schema; if schema changes are not d
 
 ```
 schema_config:
-    configs:
-        - from: "2020-07-31"
-          index:
-            period: 24h
-            prefix: loki_ops_index_
-          object_store: gcs
-          schema: v11
-          store: tsdb
-        - from: "2022-01-20"
-          index:
-            period: 24h
-            prefix: loki_ops_index_
-          object_store: gcs
-          schema: v13
-          store: tsdb
+  configs:
+    - from: "2020-07-31"
+      index:
+        period: 24h
+        prefix: loki_ops_index_
+      object_store: gcs
+      schema: v11
+      store: tsdb
+    - from: "2022-01-20"
+      index:
+        period: 24h
+        prefix: loki_ops_index_
+      object_store: gcs
+      schema: v13
+      store: tsdb
```

Comment:
Copy/ paste error?
Lines 68 - 70 are the same as lines 75 - 77, with the exception of the schema value.

Reply:
actually all this was is a change to indentation to remove the tab chars and use spaces instead which I think is better for copy pasting yaml